staging/src/k8s.io/kubectl/pkg/cmd/create/create_job.go

```diff
@@ -254,29 +256,33 @@ func (o *CreateJobOptions) createJob() *batchv1.Job {
 	return job
 }
 
-func (o *CreateJobOptions) createJobFromCronJobV1Beta1(cronJob *batchv1beta1.CronJob) *batchv1.Job {
+func (o *CreateJobOptions) createJobFromCronJobV1Beta1(cronJob *batchv1beta1.CronJob, controller bool) *batchv1.Job {
```

Comment:
Instead of changing the function, why not simply verify o.Controller inside it? :D